components/Feedback.js

```diff
@@ -84,7 +84,7 @@ export default function Feedback(className = '') {
     return (
         <div className={`flex flex-col rounded-md shadow-md border border-gray-200 p-4 ${className}`}>
             <div className='flex flex-row'>
-                <img src="/img/illustrations/icons/icon.svg" className='w-28 md:w-14' />
+                <img src="/img/illustrations/icons/icon.svg" className='w-28 md:w-14' alt="" aria-hidden="true" />
```

Comment:
why empty `alt=""`?

Reply:
I left it blank because the image looks like it's for purely decoration.

Reply:
Yes. The WCAG explanation for decorative images is to use empty alt instead of not providing it at all.
[https://www.w3.org/WAI/tutorials/images/decorative/](https://www.w3.org/WAI/tutorials/images/decorative/)

Reply:
This is the image by the way:

![Screenshot (446)](https://user-images.githubusercontent.com/70102539/229291378-f4704be4-72cf-4d50-8f98-4381e6a91cc9.png)